packages/graphql-auth-transformer/src/ModelAuthTransformer.ts

```diff
@@ -1469,6 +1480,33 @@ All @auth directives used on field definitions are performed when the field is r
         }
     }
 
+    private protectSearchQuery(ctx: TransformerContext, def: ObjectTypeDefinitionNode,
+        resolverResourceId: string, rules: AuthRule[]) {
+            const resolver = ctx.getResource(resolverResourceId);
+            if (!rules || rules.length === 0 || !resolver) {
```

Comment:
When woukd `resolver` be falsy? Should we not protect if resolver is `falsy`

Reply:
The `resolver` would be falsy if say if the respective query for searchable is set to null.
```gql
@searchable(queries: null)
```